source/examples/rapids-sagemaker-hpo/Dockerfile

```diff
@@ -10,20 +8,18 @@ ENV AWS_CV_FOLDS="10"
 # ensure printed output/log-messages retain correct order
 ENV PYTHONUNBUFFERED=True
 
-# add sagemaker-training-toolkit [ requires build tools ], flask [ serving ], and dask-ml
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential \
-    && source activate rapids \
-    && pip3 install sagemaker-training cupy-cuda11x flask dask-ml \
-    && pip3 install --upgrade protobuf
+# install a few more dependencies
+RUN conda install --yes -n base \
+        cupy \
+        flask \
+        protobuf \
+        sagemaker
 
 # path where SageMaker looks for code when container runs in the cloud
 ENV CLOUD_PATH="/opt/ml/code"
 
 # copy our latest [local] code into the container
 COPY . $CLOUD_PATH
 
-# make the entrypoint script executable
-RUN chmod +x $CLOUD_PATH/entrypoint.sh
```

Comment:
`docker build` fails at this point in the `rapids-sagemaker-hpo` notebook, because `chmod` is not permitted by default in the `rapidsai/core` image.

```text
chmod: changing permissions of '/opt/ml/code/entrypoint.sh': Operation not permitted
The command '/bin/bash -euo pipefail -c chmod +x $CLOUD_PATH/entrypoint.sh' returned a non-zero code: 1
```